Products/Zuul/infos/template.py

```diff
@@ -488,6 +488,9 @@ def getRateOptions(self):
             if self._object.isCounter():
                 rateOptions['counter'] = True
                 rateOptions["resetThreshold"] = 1
+            if self._object.rrdtype == "DERIVE" and str(self.rrdmin) != '0':
```

Comment:
Would be more logical. However it is required to post `counter = False` into JSON request to OpenTSDB to fix the issue. Other request types does not require this value to be set.